### ISSUE TITLE
Need Flutter app for our woocommerce website

### ISSUE THREAD

**Issue by rozariogem3s** [inspireui/support#3742]
This is rozario from gem3s technologies pvt ltd. We have an eCommerce solution called "kadalunavu". Kadalunavu is a woocommerce website to sell seafoods, chicken, meats, etc. We have checked your flutter app in codecanyon - https://codecanyon.net/item/fluxstore-woocommerce-flutter-ecommerce-full-app/24050041

We need to purchase your app. But we have some additional requirements needed to integrate in your flutter app. 

Is that possible to integrate additional requirements for us in the woo-commerce flutter app??

If you are interested, kindly let us know, we will share our requirements.

If you guys are interested, Kindly share your contact details (skype or whatsapp).

Awaiting your reply.

**Comment by minhcasi** [comment]
sent email contact :)

**Comment by rozariogem3s** [comment]
Our Additional Requirement for flutter mobile app. Kindly check and let me know ur comments. rozario@gem3s.in is my email id

1.Delivery date and time picker needed in the checkout billing section.
currently we are using WooCommerce Delivery Slots plugin in our website - https://iconicwp.com/products/woocommerce-delivery-slots/

2.OTP register and OTP login needed for mobile app.
currently we used Digits - WordPress Mobile Number Signup and Login plugin for OTP register and login. -https://codecanyon.net/item/digits-wordpress-mobile-number-signup-and-login/19801105

3.Payment gateway needs to be supported in mobile app. Currently we use paypal, payumoney, cashfree gateways.

4.Variable product variation must be in image not dropdown. We already used this concept in our website for the variable product section. We need the same thing for app.
Website URL:https://kadalunavu.com/app/

5.Subscribe option needed if the product is out of stock. Already we used this on the website.

6.Order notification sent via sms to customer phone number.

7.Chat needs to be integrated in app. currently we use https://www.tawk.to/ chat plugin.
